src/Component/Configuration/Loader/BabelLoader.php

```diff
@@ -28,6 +28,12 @@ public static function applyConfiguration(NodeBuilder $node_builder)
         $node_builder
             ->arrayNode('babel')
                 ->canBeDisabled()
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->arrayNode('presets')
```

Comment:
What are the defaults here?
What happens if nothing is specified? Does the plugin still work when its enabled but no presets are set?